ksql-rest-model/src/main/java/io/confluent/ksql/rest/entity/HealthcheckResponse.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.rest.entity;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.errorprone.annotations.Immutable;
+import java.util.Map;
+import java.util.Objects;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Immutable
+public final class HealthcheckResponse {
+  private final boolean isHealthy;
+  private final Map<String, HealthcheckResponseDetail> details;
+
+  @JsonCreator
+  public HealthcheckResponse(
+      @JsonProperty("isHealthy") final boolean isHealthy,
```

Comment:
Maybe change this to `isServerHealthy` to distinguish it from the same field in the response detail class. Or whatever other name you can think of, I think my suggestion sounds a little clunky.

Reply:
Why is it preferable to distinguish the two? They represent the same thing (is the resource healthy/available/up/etc) so it makes sense to me that they have the same name. I'm not particularly found of `isHealthy` for either, though. Happy the change it if there's something you like better.

Reply:
I can't really think of anything better. We can leave this as is then.